renderer/BUILD.gn

```diff
@@ -117,6 +117,22 @@ impeller_component("renderer") {
   frameworks = [ "Metal.framework" ]
 }
 
+shared_library("tesspeller") {
```

Comment:
Nit: Can this just be called tessellator please?

Reply:
Awww ok.

For how it's used, see https://github.com/jonahwilliams/svg_2/blob/main/packages/compiler/lib/src/tesspeller.dart

Perhaps it would make more sense to have that FFI wrapper live in this repo as well, at least for now. But the basic idea is to have a way to get the vertices from a path definition.